src/components/settings/Settings.h

```diff
@@ -38,12 +38,14 @@ namespace Pinetime {
         Pink
       };
       enum class PTSGaugeStyle : uint8_t { Full, Half, Numeric };
+      enum class PTSWeather : uint8_t { On, Off };
```

Comment:
Is there a third possible value in the near future or could it just be a `bool`?

Reply:
it could be a bool currently, I'm not sure if/when more options will be available but since this only a first approach I do expect it to change in the future. What's the cost/benefit of bool vs enum?

Reply:
`bool` should have the same size as `uint8_t` do. That's not the problem. But using `bool` would simplify the `...::Get() == ...::On` comparisons.

Reply:
I think I'd rather leave it as-is if there's no performance impact, for consistency with the other settings and future options.